lambdas/backend/express-route-handlers.js

```diff
@@ -418,6 +414,49 @@ async function deleteAdminCatalogVisibility(req, res) {
     }
 }
 
+async function idempotentSdkGenerationUpdate(parity, id, res) {
```

Comment:
Maybe a JSDoc here? `parity` is a confusing name.

Reply:
Yeah - very much expecting to get JSDocs and tests here shortly, but it was functional, so I wanted to give Nitish something to build against. :)

Reply:
Got it! Thanks! 😁